### DISCUSSION TITLE
How to reduce height of tab bar, navbar, bookmark bar

### DISCUSSION THREAD

**Discussion by dovbaeb** [adriankarlen/textfox#59] [Q&A] [answered]
I've looked into the css but can't seem to find any option there, even randomly playing around with variables has given nothing

**Comment by adriankarlen** [comment, marked as answer]
Yeah, this is not a variable currently, perhaps it could be, or be made relative to the font-size perhaps. At the moment they are not all always the same, it could perhaps be made relative to font-size, which is configurable. 

Change the margins and paddings of the following elements.
- URL-bar = `#navbar`
- Bookmarks = `#PersonalToolbar`
- Horizontal tabs = `#TabsToolbar`.
